utils/doclint/api_parser.js

```diff
@@ -169,30 +177,34 @@ class ApiParser {
       if (!arg)
         return;
       arg.name = name;
-      const existingArg = method.argsArray.find(m => m.name === arg.name);
-      if (existingArg && isTypeOverride(existingArg, arg)) {
-        if (!arg.langs || !arg.langs.only)
-          throw new Error('Override does not have lang: ' + spec.text);
-        for (const lang of arg.langs.only) {
-          existingArg.langs.overrides = existingArg.langs.overrides || {};
-          existingArg.langs.overrides[lang] = arg;
+      for (const method of methods) {
+        const existingArg = method.argsArray.find(m => m.name === arg.name);
+        if (existingArg && isTypeOverride(existingArg, arg)) {
+          if (!arg.langs || !arg.langs.only)
+            throw new Error('Override does not have lang: ' + spec.text);
+          for (const lang of arg.langs.only) {
+            existingArg.langs.overrides = existingArg.langs.overrides || {};
+            existingArg.langs.overrides[lang] = arg;
+          }
+        } else {
+          method.argsArray.push(arg);
         }
-      } else {
-        method.argsArray.push(arg);
       }
     } else {
       // match[1] === 'option'
-      const p = this.parseProperty(spec, match[2]);
-      if (!p)
-        return;
-      let options = method.argsArray.find(o => o.name === 'options');
-      if (!options) {
-        const type = new docs.Type('Object', []);
-        options = docs.Member.createProperty({ langs: {}, since: method.since, deprecated: undefined, discouraged: undefined }, 'options', type, undefined, false);
-        method.argsArray.push(options);
+      for (const method of methods) {
+        const p = this.parseProperty(spec, match[2]);
```

Comment:
This parses the property again and again, I guess to put a different instances into each `method`? If so, the previous block of code does not do that and puts a single `arg` into multiple `method`s - perhaps a mistake?

Reply:
Good catch. I originally intended for the parsing to exist outside of the loop (not do duplicate work), but I agree that it's a better idea to keep separate instances of each arg. I replicated this in the other site and added a comment that it's on purpose.